posthog/utils.py

```diff
@@ -631,8 +631,10 @@ class GenericEmails:
 
     def __init__(self):
         with open(get_absolute_path("helpers/generic_emails.txt"), "r") as f:
-            dd = [x.rstrip() for x in f]
-            self.emails = dd
+            self.emails = {x.rstrip(): True for x in f}
```

Comment:
Just nitpicking since performance is basically the same, but considering the intentions `set` (with set comprehension: `{domain.rstrip() for domain in f}` would be a bit clearer.